### DISCUSSION TITLE
Show time tracked via CU itself?

### DISCUSSION THREAD

**Discussion by brendanjones** [gwleuverink/clickup-time-tracker#27] [Ideas]
Is the Time Tracker app supposed to show time that's been tracked through the Clickup app? If not then this is a feature request: it'd be great to see any time already tracked in Clickup so you can fill in the gaps with this app.

Thanks for your work 🙏

**Comment by gwleuverink** [comment]
Hi Brendan,

Yes the app pulls tracked time directly from ClickUp. 

In order to view it correctly though it is important to log your hours with the correct start time in ClickUp, otherwise all hours will be logged on the same slot (when for example logging all hours at the end of the day)

**Reply by gwleuverink** [reply]
You can drag them in place in the app. But if a lot of entries overlap it becomes a bit of a squeeze on smaller screens

**Reply by brendanjones** [reply]
Ah, I didn't see my tracked time because it was displaying at weird times. Maybe a bug?

There seems to be a consistent difference between my Clickup start/end time and when it's displayed in this app. If I enter time tracking for 17:00-19:00 in CU, it displays at 09:30-11:30 in this app.

**Reply by brendanjones** [reply]
btw I was just experimenting with Manual entry; if I put in a manual entry for yesterday it appears to be entirely random where in the day it puts my time. One time it put it starting at 20:08, another at 17:15, and so on. Any ideas the rules that govern that? (Just curious, if you know it then grand. Don't feel you need to dig into this or anything, you're not Clickup Support ;) )

**Reply by brendanjones** [reply]
Okay not a bug, unless it's on Clickup's side: my timezone was changed. I've changed it back to my timezone and everything's consistent between CU and Time Tracker. This discussion can be closed.

**Reply by gwleuverink** [reply]
Hi Brendan, The app detects the timezone your computer is set to. ClickUp however uses the timezone set inside your account's preferences. It could be these two were out of sync? I'm glad it works for you now! Let me know if you run into anything else
